content/en/docs/how-tos/envelope-object.md

```diff
@@ -91,14 +128,37 @@ spec:
     kind: Namespace
     name: app
     version: v1
+  - group: placement.kubernetes-fleet.io
```

Comment:
Hi Ryan! Without RP this should be a ClusterResourceEnvelope as I understand?

Reply:
opps, agent didn't know that yet.